CMakeLists.txt

```
@@ -12,13 +12,10 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 add_compile_definitions(PLATFORM_CPU_SISD)
-set(CMAKE_CXX_FLAGS "-O3 -Wall -march=native -mtune=native")
-
-IF (WIN32)
-    set(patch_file ${CMAKE_CURRENT_BINARY_DIR}/_deps/lbm-src/src/core/singleton.h)
-    set(lbm_patch sed -i -f ${CMAKE_CURRENT_SOURCE_DIR}/script.sed ${patch_file})
+IF (!WIN32)
```

This check does not work for me on Linux and is always true. Using `NOT` instead of the exclamation mark seems to work - but you could also simply swap the branches to avoid the negation altogether.

Since the majority of the `if`/`else`/`endif` in this file seem to be lower-case, maybe also consider using that instead for consistency reasons?

```suggestion
if (NOT WIN32)
```